pinot-spi/src/main/java/org/apache/pinot/spi/accounting/ThreadExecutionContext.java

```diff
@@ -23,6 +23,12 @@
  */
 public interface ThreadExecutionContext {
 
+   public enum TaskType {
+      SSE, // Single Stage Engine
+      MSE, // Multi Stage Engine
+      UNKNOWN // Default
```

Comment:
nit: Comments are great, but that would be better if they were included as javadoc, so IDE users can get the same comment without having to download (and open) the sources

Reply:
Done